src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1546,6 +1558,18 @@ groovyscript.wiki.pyrotech.kiln.failureChance.value=Sets the chance to fail the
 groovyscript.wiki.pyrotech.kiln.failureOutput.value=Sets the output when the recipe failed
 groovyscript.wiki.pyrotech.kiln.add=Adds recipes in the format `name`, `input`, `output`, `burnTime`, `failureChance`, `failureOutput`
 
+groovyscript.wiki.pyrotech.stone_kiln.title=Stone Kiln
+groovyscript.wiki.pyrotech.stone_kiln.description=Converts an item into a new one by burning it. Has a chance to fail
+groovyscript.wiki.pyrotech.stone_kiln.burnTime.value=Sets the time required for the recipe to complete
+groovyscript.wiki.pyrotech.stone_kiln.failureChance.value=Sets the chance to fail the recipe
+groovyscript.wiki.pyrotech.stone_kiln.failureOutput.value=Sets the output when the recipe failed
+groovyscript.wiki.pyrotech.stone_kiln.add=Adds recipes in the format `name`, `input`, `output`, `burnTime`, `failureChance`, `failureOutput`
+
+groovyscript.wiki.pyrotech.stone_oven.title=Stone Oven
+groovyscript.wiki.pyrotech.stone_oven.description=Can cook food. Includes some recipes from the Furnace Registry that can't be removed here
```

Comment:
should add the second line as a warning, there are a few examples to use - thinking Blood Magic's Blood Altar as an example of the syntax